test/apps/aggregator/app.ts

```diff
@@ -0,0 +1,32 @@
+import {Deck, OrthographicView} from '@deck.gl/core';
+import {HistogramLayer} from './histogram-layer';
+
+new Deck({
+  views: new OrthographicView(),
+  initialViewState: {
+    target: [0, 0, 0],
+    zoom: 1
+  },
+  controller: true,
+  layers: [
+    new HistogramLayer({
+      data: generateData(10000, 0, 100),
+      getPosition: d => d,
+      gpuAggregation: true,
```

Comment:
Great example :) unfortunately with `gpuAggregation: true` I get a blank screen on M3. CPU version works

Reply:
Ah didn't realize `binOptions` has to be UBO after rebase.